src/intervals/arithmetic/trigonometric.jl

```diff
@@ -65,8 +68,8 @@ function Base.sin(x::BareInterval{T}) where {T<:AbstractFloat}
 
     lo, hi = bounds(x)
 
-    lo_quadrant = _quadrant(lo)
-    hi_quadrant = _quadrant(hi)
+    lo_quadrant = _quadrant(min, lo)
```

Comment:
Why do you need the `min` and `max` here?

Reply:
`_quadrant` may return two distinct integers. What `min`/`max` aims to do is to "overshoot" the quadrants: for the lowest (resp. largest) value of the interval, we want the smallest (resp. largest) quadrant.